logging/src/main/java/tech/pegasys/teku/logging/StatusLogger.java

```diff
@@ -98,6 +98,18 @@ public void beginInitializingChainData() {
     log.info("Initializing storage");
   }
 
+  public void initializingChainDataTimeout() {
+    log.info("storage initialisation timed out, will retry.");
+  }
```

Comment:
We don't want to log this to the console.  Just a normal trace log in `StorageBackedRecentChainData` would be enough. We'll almost certainly hit this at least once during startup because of the order that services start.